### PR TITLE
Add SECURITY.md (and MAINTAINERS.md)

### PR DESCRIPTION
**Pull request by AkihiroSuda**

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Lima maintainers
+
+| Name         | GitHub ID (not Twitter ID)                     | E-mail address for reporting security issues | GPG key ID       |
+|--------------|------------------------------------------------|----------------------------------------------|------------------|
+| Akihiro Suda | [@AkihiroSuda](https://github.com/AkihiroSuda) | (See GitHub profile)                         | 49524C6F9F638F1A |
```
Comment (jandubois) [inline MAINTAINERS.md:5]: Maybe I'm doing it wrong, but searching https://keys.openpgp.org for either `suda.kyoto@gmail.com` or `49524C6F9F638F1A` shows no results.

**Review by jandubois — APPROVED** [review]
LGTM, even though I'm confused about the GPG data.

But I rarely use GPG, so maybe I don't know what I'm doing... I guess the GPG ID is not a fingerprint.

File: MAINTAINERS.md (hunk 1 above)
Comment (AkihiroSuda) [reply MAINTAINERS.md:5]: Uploaded

**Comment by AkihiroSuda** [comment]
>  I guess the GPG ID is not a fingerprint.

Added the fingerprint

**Review by jandubois — APPROVED** [review]
LGTM

**Pull request merged**